src/locale/en.json

```diff
@@ -199,6 +199,7 @@
     "ContributePower": "Shows items that have power and can contribute to your power level.",
     "Cosmetic": "Shows items that are flair or cosmetic.",
     "Craftable": "Shows items that are craftable.",
+    "CraftedDupe": "Shows duplicate items where at least one of the duplicates is crafted.",
```

Comment:
This needs to be added to `config/i18n.json`, then `yarn i18n` will create the entry in this file.

Reply:
Thanks - I only ran `i18n` once. Whoops!